turborepo-tests/integration/tests/run-logging/verbosity.t

```diff
@@ -37,96 +35,11 @@ Verbosity level 1
-  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::run::global_hash: external deps hash: 459c029558afe716 (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::task_hash: task hash env vars for util:build (re)
-   vars: []
-  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::task_graph::visitor: task util#build hash is 1ce33e04f265f95c (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turbo: Found go binary at "[\-\w\/\.]+" (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turbo: build tag: rust (re)
-  [-0-9:.TWZ+]+ \[INFO]  turbo: skipping turbod since we appear to be in a non-interactive context (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turbo: filter patterns: patterns=\["util"] (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turbo: Parsed selector: selector="&{includeDependencies:false matchDependencies:false includeDependents:false exclude:false excludeSelf:false followProdDepsOnly:false parentDir: namePattern:util fromRef: toRefOverride: raw:util}" (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turbo: Filtering packages: allPackageSelectors=\["&{false false false false false false  util   util}"] (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turbo: Filtered packages: selector="&{includeDependencies:false matchDependencies:false includeDependents:false exclude:false excludeSelf:false followProdDepsOnly:false parentDir: namePattern:util fromRef: toRefOverride: raw:util}" entryPackages=map\[util:util] (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turbo: Filtered packages: cherryPickedPackages=map\[util:util] walkedDependencies=map\[] walkedDependents=map\[] walkedDependentsDependencies=map\[] (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turbo: Filtered packages: cherryPickedPackages=map\[] walkedDependencies=map\[] walkedDependents=map\[] walkedDependentsDependencies=map\[] (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turbo: filtered packages: packages=map\[util:util] (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turbo: global hash env vars: vars=\[] (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turbo: global hash: value=2107205f32600eb9 (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turbo: global hash matches between Rust and Go (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turbo: local cache folder: path="" (re)
-  \xe2\x80\xa2 Packages in scope: util (esc)
-  \xe2\x80\xa2 Running build in 1 packages (esc)
-  \xe2\x80\xa2 Remote caching disabled (esc)
-  [-0-9:.TWZ+]+ \[DEBUG] turbo: task hash env vars for util:build: vars=\[] (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turbo.: start (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turbo: task hash: value=1ce33e04f265f95c (re)
-  util:build: cache bypass, force executing 1ce33e04f265f95c
-  util:build: 
-  util:build: > build
-  util:build: > echo building
-  util:build: 
-  util:build: building
-  [-0-9:.TWZ+]+ \[DEBUG] turbo.: caching output: outputs="{\[packages/util/.turbo/turbo-build.log] \[]}" (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turbo.: done: status=complete duration=[\.0-9]+m?s (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turbo: task hashes match (re)
-  
-   Tasks:    1 successful, 1 total
-  Cached:    0 cached, 1 total
-    Time:\s*[\.0-9]+m?s  (re)
-  
+  $ ${TURBO} build -vv --filter=util --force 1> VERBOSEVV 2>&1
```

Comment:
Is that all that we need to test? Should we verify that the filtering/force flags are doing what they should?

Reply:
I don't think we need to for this test. We have other tests for filtering and cache usage.